xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/SuggestInputElement.java

```diff
@@ -241,7 +241,7 @@ public SuggestInputElement selectByValue(String value)
      */
     public SuggestInputElement selectByVisibleText(String text)
     {
-        getDriver().findElementWithoutWaiting(
+        getDriver().findElement(
```

Comment:
Why wait? I think this method is supposed to be called **after** the suggestions are displayed. It assumes you already waited for the suggestions. Maybe the problem is that you didn't wait for the suggestions?